main/guides/getting-started/README.md

```diff
@@ -85,7 +107,7 @@ Cloning and installing the Agoric SDK can take a while. Please be patient.
 yarn install
 ```
 
-**Note:** If you run into errors during install or build, make sure you have build-essential installed. `gcc --version`.
+**Note:** If you run into errors during install or build, make sure you have build-essential components installed. `gcc --version`.
```

Comment:
**important**:

"build-essential" here is a noun, not an adjective. It's the name of a debian / Ubuntu package. So the note should perhaps say:

```suggestion
**Note:** If you run into errors during install or build, make sure you have the relevant devloper tools installed. For example, on debian or Ubuntu linux: `sudo apt-get install build-essential`.
```

By way of example, DigitalOcean, which has really good docs, discusses it this way:

[How To Compile and Install Packages From Source Using Make on a VPS \| DigitalOcean](https://www.digitalocean.com/community/tutorials/how-to-compile-and-install-packages-from-source-using-make-on-a-vps)

> To compile sources on Linux, you will need the package called "build-essential" on Debian-based systems and "Development Tools" on CentOS, as it contains the gcc/g++ compilers and libraries required to compile packages. To install this on Debian and Ubuntu run:
> 
> ```
> apt-get install build-essential
> ```
> And on CentOS run:
>
> ```
> yum groupinstall "Development Tools"
> ```

but they don't support Mac nor WSL.

Reply:
The text has been changed to match your suggestion.